addons/sale/models/sale.py

```diff
@@ -626,6 +626,15 @@ def force_quotation_send(self):
                 order.with_context(email_ctx).message_post_with_template(email_ctx.get('default_template_id'))
         return True
 
+    @api.multi
+    def send_order_confirmation_mail(self, transaction=False):
+        # send order confirmation mail when payment done through the portal
+        confirmation_template = self.env['ir.config_parameter'].sudo().get_param('sale.sale_confirmation_template', False)
+
+        for order in self.filtered(lambda so: so.team_id.team_type != 'website'):
```

Comment:
Why this condition ? I think it has nothing to do here as it is not modular.

Reply:
@tde-banana-odoo , we have two separate email template settings for e-commerce and portal so to check either the order comes from the portal or from the e-commerce we used team_type because there is default team when the order comes from e-commerce. As team_id is in sale module so I think it's modular. (if we merge this two settings this condition should be removed) Thanks